apps/web/server/lib/[user]/getServerSideProps.ts

```diff
@@ -135,10 +138,11 @@ export const getServerSideProps: GetServerSideProps<UserPageProps> = async (cont
     image: getUserAvatarUrl({
       avatarUrl: user.avatarUrl,
     }),
-    theme: user.theme,
-    brandColor: user.brandColor ?? DEFAULT_LIGHT_BRAND_COLOR,
+    theme: user.profile.organization?.theme ?? user.theme,
+    brandColor: user.profile.organization?.brandColor ?? user.brandColor ?? DEFAULT_LIGHT_BRAND_COLOR,
     avatarUrl: user.avatarUrl,
-    darkBrandColor: user.darkBrandColor ?? DEFAULT_DARK_BRAND_COLOR,
+    darkBrandColor:
+      user.profile.organization?.darkBrandColor ?? user.darkBrandColor ?? DEFAULT_DARK_BRAND_COLOR,
```

Comment:
Can we not use the similar abstraction here. Like we could have one to get it for a user within the same modiule(getBranding)